packages/x-components/tests/e2e/cucumber/filters/clear-filters.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { And, Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+// Scenario 1
+When('clear filters button is clicked', () => {
+  cy.getByDataTest('clear-filters').click();
+});
+
+And(
+  'filters {string} are shown in the selected filters list',
+  function (this: any, clickedFiltersIndex: string) {
+    const clickedFiltersIndexList = clickedFiltersIndex.split(', ');
+    for (let i = 0; i < clickedFiltersIndexList.length; i++) {
```

Comment:
Any reason why you choose a classic `for` loop instead of a `forEach`? Just curious.

Reply:
Nope, I'll change it